isb_lib/core.py

```diff
@@ -40,8 +48,89 @@ def relationAsSolrDoc(
     doc["tstamp"] = datetimeToSolrStr(ts)
     return doc
 
+def _shouldAddMetadataValueToSolrDoc(metadata: typing.Dict, key: typing.AnyStr) -> bool:
+    shouldAdd = False
+    if key in metadata:
+        value = metadata[key]
+        if type(value) is list:
```

Comment:
Attempted to not index empty fields.  If you can think of additional type-specific checks we should be doing here, please fire away!

Reply:
Could perhaps add a range check for lat lon values, but otherwise this seems fine. I presume strings are stripped of leading and trailing whitespace earlier?

Reply:
Actually, we aren't doing that.  Sounds like we should.  Easy enough to add, I can do that.

Reply:
Implemented both suggestions, please take a look at latest version.

Reply:
Looks good.